teacher.rb

```diff
@@ -0,0 +1,16 @@
+require 'person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown', permission: true)
+    super(age, name, permission)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+
+  def play_hooky
+    puts '¯\(ツ)/¯'
+  end
```

Comment:
- [ OPTIONAL BUT HIGHLY RECOMMENDED ] Hello! According to the project guidelines, there's no need to include the `play_hooky` method inside the Teacher class. 📜🚫 Please remove it from this class as it deviates from the project guidelines. 🛠️ This ensures that our code aligns with the prescribed project requirements and maintains consistency. 👍